Robot2019/src/main/java/frc/robot/OI.java

```diff
@@ -83,7 +83,7 @@
     climberRailBtn.whenPressed(new ToggleClimberRails(climber));
 
     autoClimbBtn = new JoystickButton(manipulator, Manip.RT_rTrigger);
-    autoClimbBtn.toggleWhenPressed(new Climb(climber, dt, leftJoy));
+    autoClimbBtn.toggleWhenPressed(new Climb(climber, dt, leftJoy, lights));
 
     manualClimbBtn = new JoystickButton(manipulator, Manip.LT_lTrigger);
     manualClimbBtn.toggleWhenPressed(new ManualClimb(climber, manipulator));
```

Comment:
Same changes should be made to ManualClimb. I don't think the Climb command has actually been tested yet.